jupyter_rsession_proxy/__init__.py

```diff
@@ -102,13 +108,27 @@ def _get_cmd(port):
         ]
         # Support at least v1.2.1335 and up
 
-        if _support_arg('www-root-path'):
+        supported_args = _support_args([
+            'www-root-path',
+            'server-data-dir',
+            'database-config-file',
+            'www-thread-pool-size',
+        ])
+        if supported_args['www-root-path']:
             cmd.append('--www-root-path={base_url}rstudio/')
-        if _support_arg('server-data-dir'):
+        if supported_args['server-data-dir']:
             cmd.append(f'--server-data-dir={server_data_dir}')
-        if _support_arg('database-config-file'):
+        if supported_args['database-config-file']:
             cmd.append(f'--database-config-file={database_config_file}')
 
+        if supported_args['www-thread-pool-size']:
+            try:
+                thread_pool_size = int(os.getenv('RSERVER_THREAD_POOL_SIZE', ""))
+                if thread_pool_size > 0:
+                    cmd.append('--www-thread-pool-size=' + str(thread_pool_size))
+            except Exception:
+                pass
+
```

Comment:
If this is errorously configured, such as having `RSERVER_THREAD_POOL_SIZE` set to "asdf", I think it can make sense to error loudly instead. Of course, if its an empty string though, we shouldn't error, but we can check that without a try/except block.